src/game_actor.h

```diff
@@ -827,8 +828,8 @@ class Game_Actor final : public Game_Battler {
 
 	int GetBattleAnimationId() const override;
 
-	int GetHitChance() const override;
-	float GetCriticalHitChance() const override;
+	int GetHitChance(int weapon = kWeaponAll) const override;
+	float GetCriticalHitChance(int weapon = kWeaponAll) const override;
```

Comment:
> Allow computing stats as if single weapon equipped

Sorry, but for all of them the doc comment was not updated.

From ``GetBaseAtk`` to ``AttackIgnoresEvasion``

Also in ``Game_battler.h`` and ``game_enemy.h``

Reply:
Added